tests/Test/Xrefcheck/UtilRequests.hs

```diff
@@ -0,0 +1,75 @@
+{- SPDX-FileCopyrightText: 2019 Serokell <https://serokell.io>
+ -
+ - SPDX-License-Identifier: MPL-2.0
+ -}
+
+module Test.Xrefcheck.UtilRequests
```

Comment:
I have refactored some functions from `Test.Xrefcheck.TooManyRequestsSpec` in order to reuse them in `Test.Xrefcheck.RedirectRequestsSpec`.